data/scripts/packethandlers/core.lua

```diff
@@ -0,0 +1,13 @@
+local creatureevent = CreatureEvent("NetworkMessage")
+
+function creatureevent.onNetworkMessage(player, recvByte, msg)
+	local handler = PacketHandlers[recvByte]
+	if not handler then
+		io.write(string.format("Player: %s sent an unknown packet header: 0x%02X with %d bytes!\n", player:getName(), recvByte, msg:len()))
+		return
+	end
+
+	handler(player, msg)
+end
+
+creatureevent:register()
```

Comment:
I don't like this file being here, it's error prone and server owners might inadvertently remove it and lose the entire functionality. However, I can't register the creature event inside the lib (`g_creatureEvents` is a null pointer and it crashes), and I didn't want to add a `PacketHandler` event + `PacketHandler` base events to deal with it. Ideas?